client/src/views/page-components/MainPage/ChatListScreen/ChatListScreen.js

```diff
@@ -0,0 +1,18 @@
+import EMPTY_ERROR_MESSAGE from '../../../../constants/emptyErrorMessage.js';
+import Product from '../../../components/Product/Product.js';
+import Thung from '../../../components/Thung/Thung.js';
+
+export default function ChatListScreen({ $selector, chatList }) {
+  this.render = () => {
+    if (chatList.length === 0) {
+      return new Thung({
+        $selector: document.querySelector('.menu-slide-main'),
+        message: EMPTY_ERROR_MESSAGE['chatList'],
+      });
```

Comment:
함수이기 때문에 return을 해주시는 걸까요??
위의 render 함수에서 return이 없어서, 
return이 필요 없을 때의 상황을 통일하는 게 좋지 않을까라는 생각을 해봅니다!..

Reply:
함수라기보다는 chatList가 [] 일 때 아래의 코드를 실행하지 못하게 return 하는 용도로 사용했습니다
이게 의미적으로 더 맞겠네요
```suggestion
      new Thung({
        $selector: document.querySelector('.menu-slide-main'),
        message: EMPTY_ERROR_MESSAGE['chatList'],
      });
      return;
```